KarhooUISDK/Screens/BookingRequestScreen/FormBookingRequestScreen/CheckoutViews/MoreDetailsView.swift

```diff
@@ -17,6 +17,7 @@ struct KHMoreDetailsViewID {
 final class MoreDetailsView: UIView {
     
     private var didSetupConstraints: Bool = false
+    private var maximumNumberOfViewsReached: Int = 0
```

Comment:
This is not used at the moment. Will it be used in a future commit?

Reply:
will remove it now, it was used in a initial implementation and forgotten when cleaning up the code